lib/cassandra_migrations/railtie/tasks.rake

```diff
@@ -65,10 +67,10 @@ namespace :cassandra do
   end
 
   namespace :test do
-    desc 'Load the development schema in to the test keyspace'
+    desc 'Load the development schema in to the test keyspace via a full reset'
     task :prepare do
       Rails.env = 'test'
-      Rake::Task['cassandra:reset'].execute
+      Rake::Task['cassandra:migrate:reset'].execute
```

Comment:
Yeah, I think an alias will do good here. I mean, it's nothing serious, but it is always a good practice to avoid incompatibilities in minor version bumps (even if we have done this on the past, shame on us!)